### PR TITLE
916 menu item override refactor

### PR DESCRIPTION
**Pull request by ricardovdheijden**
#916 

Small refactor on the filtering of menu items

### CODE REVIEW

File: packages/orchestrator-ui-components/src/components/WfoPageTemplate/WfoSidebar/utils/menuItemIsAllowed.spec.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    ['/testUrl1', '/testPolicyResource1'],
+    ['/testUrl2', '/testPolicyResource2'],
+]);
+
+describe('menuItemIsAllowed', () => {
+    beforeEach(() => {
+        isAllowedHandlerMock.mockClear();
+    });
+
+    it('returns true when url is undefined', () => {
```
Comment (mrijk) [inline packages/orchestrator-ui-components/src/components/WfoPageTemplate/WfoSidebar/utils/menuItemIsAllowed.spec.ts:14]: Maybe it.each to have parametrized tests and reduce the amount of test code?

File: packages/orchestrator-ui-components/src/components/WfoPageTemplate/WfoSidebar/utils/menuItemIsAllowed.spec.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+        const result = menuItemIsAllowed(url, testUrlMap, isAllowedHandlerMock);
+
+        expect(result).toEqual(false);
+        expect(isAllowedHandlerMock).toHaveBeenCalledWith(
+            '/testPolicyResource1',
+        );
+    });
+
+    it('returns true url is not in the url-map', () => {
```
Comment (DutchBen) [inline packages/orchestrator-ui-components/src/components/WfoPageTemplate/WfoSidebar/utils/menuItemIsAllowed.spec.ts:40]: Typo? 
```suggestion
    it('returns true if url is not in the url-map', () => {
```

**Review by DutchBen — APPROVED** [review]

File: packages/orchestrator-ui-components/src/components/WfoPageTemplate/WfoSidebar/utils/menuItemIsAllowed.spec.ts (hunk 1 above)
Comment (DutchBen) [reply packages/orchestrator-ui-components/src/components/WfoPageTemplate/WfoSidebar/utils/menuItemIsAllowed.spec.ts]: Doesn't make it more readable I think.

**Review by mrijk — APPROVED** [review]
LGTM

**Pull request merged**